### PR TITLE
Fix logo url for invalid certificate EDLY_2440

### PR DESCRIPTION
**Pull request by taimoor-ahmed-1**
**Description:** Fix logo src url in header for invalid certificate.

**JIRA:**
https://edlyio.atlassian.net/browse/EDLY-2440

### CODE REVIEW

File: openedx/features/edly/utils.py [hunk 1]
```diff
@@ -363,7 +363,7 @@ def get_current_site_invalid_certificate_context(default_html_certificate_config
         return context
 
     context['platform_name'] = current_site_configuration.get_value('platform_name', settings.PLATFORM_NAME)
-    context['logo_src'] = get_logo_url()
+    context['logo_src'] = current_site_configuration.get_value('logo_image_url', '')
```
Comment (zubair-arbi) [inline openedx/features/edly/utils.py:366]: @taimoor-ahmed-1 @mamjad00 Are we always going to provide absolute URL's for `logo_image_url` in site configuration?
Also is this information provided in the cloud client onboarding document?

**Review by zubair-arbi — CHANGES_REQUESTED** [review]
Changes looks good. I just have one question about `logo_image_url`.

File: openedx/features/edly/utils.py (hunk 1 above)
Comment (taimoor-ahmed-1) [reply openedx/features/edly/utils.py:366]: @zubair-arbi I looked up in the onboarding document and it doesn't contain `logo_image_url` in site configurations. 
Alternatively, should we use `logo` from `BRANDING` which is already available as absolute URL?
Also, there is a related bug with `valid` certificates which should be addressed: https://edlyio.atlassian.net/browse/EDLY-2440?focusedCommentId=25777

**Review by zubair-arbi — APPROVED** [review]
Changes looks good to me. 👍

File: openedx/features/edly/utils.py [hunk 2]
```diff
@@ -363,7 +363,7 @@ def get_current_site_invalid_certificate_context(default_html_certificate_config
         return context
 
     context['platform_name'] = current_site_configuration.get_value('platform_name', settings.PLATFORM_NAME)
-    context['logo_src'] = get_logo_url()
+    context['logo_src'] = current_site_configuration.get_value('BRANDING', {}).get('logo')
```
Comment (asadmanzoorbhatti) [inline openedx/features/edly/utils.py:366]: define default value in `get('logo')`

**Review by asadmanzoorbhatti — CHANGES_REQUESTED** [review]

File: openedx/features/edly/utils.py (hunk 2 above)
Comment (taimoor-ahmed-1) [reply openedx/features/edly/utils.py]: Done

**Review by asadmanzoor93 — APPROVED** [review]
LGTM👍

**Pull request merged**